app_flutter/lib/index_page.dart

```diff
@@ -83,76 +83,65 @@ class Index extends StatelessWidget {
 
   final GlobalKey<ScaffoldState> scaffoldKey;
 
+  static const Widget separator = SizedBox(height: 24.0);
+
   @override
   Widget build(BuildContext context) {
+    final ThemeData theme = Theme.of(context);
     return Consumer<IndexState>(
-      builder: (_, IndexState indexState, Widget child) => Scaffold(
+      builder: (BuildContext context, IndexState indexState, Widget child) => Scaffold(
```

Comment:
Q: Is it better practice to leave the variable in instead of ignoring them?

Reply:
In general I would encourage it just because it makes it easier to read the code (you don't have to wonder what the argument is), but in this case I added it back just because I needed it. :-)

See also: https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo#avoid-anonymous-parameter-names

Reply:
(to clarify: i needed it, then later took out the code that needed it)